runtime/webpack.config.js

```diff
@@ -1,28 +0,0 @@
-/**
```

Comment:
It might be clearer to do this in a separate commit (although I wouldn't unwind this cl to get it out).